agent/taskresource/cgroup/cgroup.go

```diff
@@ -32,10 +32,11 @@ import (
 )
 
 const (
-	memorySubsystem         = "/memory"
-	memoryUseHierarchy      = "memory.use_hierarchy"
-	rootReadOnlyPermissions = os.FileMode(400)
-	resourceName            = "cgroup"
+	memorySubsystem           = "/memory"
+	memoryUseHierarchy        = "memory.use_hierarchy"
+	rootReadOnlyPermissions   = os.FileMode(400)
+	resourceName              = "cgroup"
+	resourceProvisioningError = "TaskResourceError: Agent could not create task's platform resources"
```

Comment:
:%s/TaskResourceError/CgroupError/ ?

Reply:
hm my thinking was it wouldn't be a customer resource, but bucketing by type is a good idea.

Reply:
"TaskResourceError" wouldn't make sense for a customer yet as they are not directly creating these resources. That is still an internal implementation detail, which is why I think "CgroupError" makes more sense in relaying what went wrong to customers in the `DescribeTasks` API response.